actions/views/rhs.js

```diff
@@ -42,11 +42,12 @@ export function updateRhsState(rhsState, channelId) {
 
 export function selectPostFromRightHandSideSearch(post) {
     return async (dispatch, getState) => {
-        await dispatch(PostActions.getPostThread(post.id));
+        const postRootId = Utils.getRootId(post);
+        await dispatch(PostActions.getPostThread(postRootId));
```

Comment:
Should we only do this when we don't have the thread already? A little worried about the number of extra requests this will create